packages/kbn-es/src/install/archive.js

```diff
@@ -107,4 +124,13 @@ async function configureKeystore(installPath, password, log = defaultLog, bundle
     cwd: installPath,
     env,
   });
+
+  for (const [secureSettingName, secureSettingValue] of secureSettings) {
+    log.info(`setting secure setting [${secureSettingName}] to %s`, chalk.bold(secureSettingValue));
```

Comment:
__note:__ `setting .... setting` not sure how that sounds :man_shrugging: Couldn't find a better verb.

Reply:
nit, but could we change this to:

```suggestion
    log.info(`setting secure setting %s to %s`, chalk.bold(secureSettingName), chalk.bold(secureSettingValue));
```